### PR TITLE
Adnow bidder

### PR DESCRIPTION
**Pull request by vingood**
## Type of change

- [ ] Bugfix
- [ ] Feature
- [x] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change
New bidder adapter: adnow.

- test parameters for validating bids
```
{
  bidder: 'adnow',
  params: {
    codeId: 794934
  }
}
```

Be sure to test the integration with your adserver using the [Hello World](/integrationExamples/gpt/hello_world.html) sample page.

- support@adnow.com
- [x] official adapter submission

For any changes that affect user-facing APIs or example code documented on http://prebid.org, please provide:

- A link to a PR on the docs repo at https://github.com/prebid/prebid.github.io/pull/2314

### CODE REVIEW

**Review by jsnellbaker — CHANGES_REQUESTED** [review]
Hi @vingood 

When I trying to test the bidder (using the params in the description), I was getting a bid response from the server but the cpm value was set to `0`.  This was causing the bid to be rejected by the logic you have in the `interpretResponse` function.  

Can you update the price on the test bid to a higher value so I can test the end-to-end workflow?

Thanks!

**Comment by vingood** [comment]
Please review again. Thank you.

**Review by jsnellbaker — CHANGES_REQUESTED** [review]
@vingood 
Thanks for making the update on the test bid data.  

However, I noticed that the `width` and `height` properties are not being defined properly in the bid object created for Prebid.  This is causing the banner ad to not physically appear on the page since it has a default `width` and `height` of `0x0`.

I see the properties are defined in the raw response from the ad server.  Can you add the logic in the adapter code to read these properties and assign them to the bid object?

Thanks.

File: test/spec/modules/adnowBidAdapter_spec.js [hunk 1]
```diff
@@ -0,0 +1,161 @@
+import { expect } from 'chai';
+import { spec } from 'modules/adnowBidAdapter.js';
+
+describe.only('adnowBidAdapter', function () {
```
Comment (jsnellbaker) [inline test/spec/modules/adnowBidAdapter_spec.js:4]: Please remove the `.only` from your describe function here.

File: test/spec/modules/adnowBidAdapter_spec.js [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+        }
+      }];
+
+      const req = spec.buildRequests(bidRequestData);
+      const reqData = req[0].data;
+
+      expect(reqData).to.match(/sizes=300x250/);
+    });
+
+    it.only('Native sizes', function () {
```
Comment (jsnellbaker) [inline test/spec/modules/adnowBidAdapter_spec.js:62]: Please remove the `.only` from your it function here as well.

**Review by jsnellbaker — APPROVED** [review]
Thanks for making the additional updates.  LGTM

**Pull request merged**